client/package.json

```diff
@@ -51,13 +51,15 @@
     "src"
   ],
   "dependencies": {
+    "file-saver": "^1.3.3",
     "inversify": "^3.1.0",
     "snabbdom": "^0.6.4",
     "snabbdom-jsx": "^0.3.1",
     "snabbdom-virtualize": "^0.6.0"
   },
   "devDependencies": {
     "@types/chai": "^3.4.35",
+    "@types/file-saver": "https://registry.npmjs.org/@types/file-saver/-/file-saver-0.0.1.tgz",
```

Comment:
This is what `npm install @types/file-saver --save-dev` produced. No idea what kind of problem that is.